tskv/src/reader/column_group/mod.rs

```diff
@@ -330,11 +312,12 @@ mod tests {
             Field::new("c4", DataType::Boolean, true),
         ]));
 
-        let column_group_reader = ColumnGroupReader::new_with_unchecked(
```

Comment:
why clippy warning this?

Reply:
This function is only used once in the test module.
Modify to the following format to reduce the amount of code：
```
        let column_group_reader = ColumnGroupReader {
            column_group: Arc::new(ColumnGroup::new(0)),
            page_readers,
            schema,
            metrics: Arc::new(ExecutionPlanMetricsSet::new()),
        };
```